test/TEST-20-MAINPIDGAMES/test.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+set -e
```

Comment:
When the `set -e` command was added into integration tests, it came with several other modifications to avoid unnecessary errors (e.g. test fail when trying to unmount an already unmounted mount). See https://github.com/systemd/systemd/commit/818567fce693519eda9d7550a9ae21a7fc974b3c#diff-7165e29cecec079eaf525e5eb910014b

Anyway, for this particular test case, this part of the aforementioned patch should be enough:
```patch
 test_cleanup() {
-    umount $TESTDIR/root/var 2>/dev/null
+    [ -d $TESTDIR/root/var ] && mountpoint $TESTDIR/root/var && umount $TESTDIR/root/var
     [[ -b /dev/mapper/varcrypt ]] && cryptsetup luksClose /dev/mapper/varcrypt
-    umount $TESTDIR/root 2>/dev/null
+    umount $TESTDIR/root 2>/dev/null || true
     [[ $LOOPDEV ]] && losetup -d $LOOPDEV
     return 0
 }
```

Reply:
Hm, maybe the right thing here is to drop the 'set -e'? The only difference I see in `test_setup` compared to the existing tests is `|| return 1`, which is obviously related to it...

Reply:
That's an option too

Reply:
Done.